### DISCUSSION TITLE
Source code modding support on Steam Workshop

### DISCUSSION THREAD

**Discussion by NoeelMoeskops** [FakeFishGames/Barotrauma#7028] [Ideas and suggestions]
Hi,

It would be great if Barotrauma supported source code mod sharing on the steam workshop. Currently, it is possible to modify the source code and compile a `Barotrauma.dll` file, but distributing the mod comes down to sending the `Barotrauma.dll` file to friends and instructing them to override it in their game directory. Also, combining multiple mods using this method is not possible.
See #7025

**Comment by iridescent-lightning** [comment]
Try LuaForBarotrauma + CsForBarotrauma

**Reply by G-M-twostay** [reply]
this doesn't solve all issues. being able to directly modify source code is on another level in terms of flexibility, simplicity, and optimizations.
